compiler/codegen/symbol.cpp

```diff
@@ -729,7 +729,9 @@ void VarSymbol::codegenDef() {
       }
     }
     if(debug_info){
-      debug_info->get_variable(this);
+      LLVM_DIVARIABLE lVDebug = debug_info->get_variable(this);
+      //create llvm.dbg.declare for this local variable
+      debug_info->createDeclare(varAlloca, lVDebug);
```

Comment:
Yea, it'll need another two parameters to call llvm::insertDeclare(). But for some reason, 3.7 is still not working properly with -g